model/facade/action/GetPersonalSummaryByLoginDateAction.php

```diff
@@ -194,12 +177,11 @@ protected function doExecute() {
         if( $this->currentJourney ) {
             $extraGoalHoursSet = 0;
             if ( $this->currentUserGoal ) {
-                // Only if some goal is set for this period
-                $weeksInBetween = $this->getWeeksInBetweenDates( $this->currentUserGoal->getInitDate(), $this->currentUserGoal->getEndDate() );
-                $extraGoalHoursSet += ( $this->currentUserGoal->getExtraHours() / $weeksInBetween );
+                $extraGoalHoursSet += $this->currentUserGoal->getExtraHours() / $this->getWeeksTillEndOfJourneyPeriod();
             }
-            $originalHoursToBeWorked = round(($this->getWorkableHoursInThisJourneyPeriod() - $this->getWorkedHoursInThisJourneyPeriod())/ $this->getWeeksTillEndOfJourneyPeriod() , 2);
-            $totalResults['weekly_goal'] = floor( ( $originalHoursToBeWorked + $extraGoalHoursSet ) * 60);
+
+            $originalHoursToBeWorked = ($this->getWorkableHoursInThisPeriod() - $this->getWorkedHoursInThisPeriod())/ $this->getWeeksTillEndOfJourneyPeriod();
+            $totalResults['weekly_goal'] = round( ( $originalHoursToBeWorked + $extraGoalHoursSet ) * 60);
```

Comment:
You are calling `$this->getWeeksTillEndOfJourneyPeriod()` twice in this small piece of code. Let's call it only once and save it to a variable so we save one call!